tests/secret.test.ts

```diff
@@ -50,131 +67,234 @@ describe("secret Tests", () => {
     client2.setAuthKey(res2.authKey);
   });
 
-  describe("create secret", () => {
-    it("create secret succesfully", async () => {
-      const secret = new Secret({
-        name: `dummysecret_${getNextId()}`,
+  describe("create secret suite", () => {
+    it("created secret have value in workflow", async () => {
```

Comment:
This newly added tests fails, but the rest are passing. The goal is to verify that the secret variable’s value show up at nodes, e.g. a CustomCode node.